RocketChat/Rocket.Chat
# fix: User Mentioning when prepending the username with >

## Proposed changes (including videos or screenshots)


This pull request fixes the irregular behavior of mentioning the user when prepending the username with ">" symbol.


https://user-images.githubusercontent.com/89914602/231296116-c057e093-7f3d-4e49-bb29-05a54d7baef4.mp4





## Issue(s)

fix #28846 
## Steps to test or reproduce

1. Type test > @any-username.
2. Notice it is getting mentioned now.
